cmd/landscaper-agent/app/app.go

```diff
@@ -43,7 +40,7 @@ func NewLandscaperAgentCommand(ctx context.Context) *cobra.Command {
 }
 
 func (o *options) run(ctx context.Context) error {
-	o.log.Info(fmt.Sprintf("Start Landscaper Agent with version %q", version.Get().String()))
+	o.log.Info("Starting Landscaper Agent", "version", version.Get().String())
```

Comment:
Is there a reason why you do not use lc.KeyVersion here?

Reply:
No, maybe that was one of the first changes before the `KeyVersion` field was added. I will update it.